integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/CoinbaseConvertCryptoFragment.kt

```diff
@@ -177,9 +263,31 @@ class CoinbaseConvertCryptoFragment : Fragment(R.layout.fragment_coinbase_conver
             binding.youWillReceiveLabel.isVisible = hasBalance
             binding.youWillReceiveValue.isVisible = hasBalance
             if (hasBalance) {
-                binding.youWillReceiveValue.text = context?.getString(R.string.you_will_receive_dash, dashFormat.format(balance).toString())
+                binding.youWillReceiveValue.text = context?.getString(
```

Comment:
You can use `requireContext` here.
```suggestion
                binding.youWillReceiveValue.text = requireContext().getString(
```